packages/table/src/table.tsx

```diff
@@ -491,6 +491,74 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     // ================
 
     /**
+     * __Experimental!__ Resizes all rows in the table to the approximate
+     * maximum height of wrapped cell content in each row. Works best when each
+     * cell contains plain text of a consistent font style (though font style
+     * may vary between cells). Since this function uses approximate
+     * measurements, results may not be perfect.
+     *
+     * Approximation parameters can be configured for the entire table or on a
+     * per-cell basis. Default values are fine-tuned to work well with default
+     * Table font styles.
+     *
+     * @param getCellText the text content of the cell
+     * @param getApproximateCharWidth the approximate width (in pixels) of an
+     * average character of text
+     * @param getApproximateLineHeight the approximate height (in pixels) of an
+     * average line of text
+     * @param getCellHorizontalPadding the sum of horizontal paddings (in
+     * pixels) from the left __and__ right sides of the cell
+     * @param getNumBufferLines the number of extra lines to add in case the
+     * calculation is imperfect
+     */
+    public resizeRowsByApproximateHeight(
+        getCellText: ICellMapper<string>,
+        getApproximateCharWidth: number | ICellMapper<number> = 8,
```

Comment:
Do we need constants for these numbers? They're already fairly well documented by their respective param names and in the function description above.

Reply:
only used here so I'm good 👍